### ISSUE TITLE
UI to set individual device settings

### ISSUE THREAD

**Issue by carandraug** [microscope-cockpit/cockpit#901]
Currently, the only devices that expose the settings dict in cockpit are cameras through their settings button on the main window:

![Screenshot from 2024-08-21 13-41-57](https://github.com/user-attachments/assets/c46fbfe6-cd6f-4488-b549-5c741846fba9)

But all Python-Microscope devices have a settings dict. Would be nice to expose that but not all devices have space for them on the main window. I'm thinking the simplest thing would be to add a menu bar for devices that would open those as separate windows.  (requested by @LauraA-Octopus on image.sc https://forum.image.sc/t/adding-settings-on-stage-controller/100713)

**Comment by carandraug** [comment]
I have implemented a minimal solution on my `901-devices-menu` branch:

![Screenshot from 2024-08-21 13-47-12](https://github.com/user-attachments/assets/3b54921f-2e18-47a9-89a0-b6942fe0a060)

This does not filter devices that are not Python-Microscope devices. We probably could group them by type somehow. And the "setings" button beneath the camera does some extra things that I'm not sure what they are for. But this is a start for people to comment and maybe use as hint to implement better.

**Comment by iandobbie** [comment]
Seems like a good idea.

**Comment by LauraA-Octopus** [comment]
This seems indeed a good idea, I can try it in the lab probably this Friday. Hopefully this will solve the issue, otherwise I will have to write a custom class for the AMC in cockpit, which I'd rather avoid since you are trying to be not device specific

**Comment by juliomateoslangerak** [comment]
Good idea. I might consider renaming the menu to Settings or Device settings though.
